src/Uno.UWP/UI/Composition/CompositionSurfaceBrush.cs

```diff
@@ -19,134 +18,93 @@ public partial class CompositionSurfaceBrush : CompositionBrush
 		private float _horizontalAlignmentRatio;
 		private bool _snapToPixels;
 		private float _verticalAlignmentRatio;
-		private CompositionBitmapInterpolationMode bitmapInterpolationMode;
+		private CompositionBitmapInterpolationMode _bitmapInterpolationMode;
 
-		internal event Action? PropertyChanged;
-
-		public CompositionSurfaceBrush(Compositor compositor) : base(compositor)
+		internal CompositionSurfaceBrush(Compositor compositor) : base(compositor)
 		{
 		}
 
-		public CompositionSurfaceBrush(Compositor compositor, ICompositionSurface surface) : base(compositor)
+		internal CompositionSurfaceBrush(Compositor compositor, ICompositionSurface surface) : base(compositor)
```

Comment:
Will be flagged as breaking, make sure to add the entry in `PackageDiffIgnore.xml`.

Reply:
Done